blocks/Xcos/blocks/CLOCK_c.py

```diff
@@ -102,21 +113,21 @@ def CLOCK_c(outroot, attribid, ordering, geometry, parameters):
                  y="40.0")
     addmxPointNode(gemotryNode, 'mxPoint',
                    a="targetPoint", x="20.0", y="-4.0")
-    CCLink = addLink(root, TYPE_LINK, id="-63efee48:189fd5ed04e:-73d4",
-                     parent="128c18ea:1383ab8277e:-748d",
-                     source="-63efee48:189fd5ed04e:-73d7",
-                     target="-63efee48:189fd5ed04e:-73dd",
+    CCLink = addLink(root, TYPE_LINK, id=link_id[1],
+                     parent=link_id[3],
+                     source=port_id[4],
+                     target=port_id[0],
                      style="CommandControlLink", value="")
     gemotryNode = addGeoNode(CCLink, GEOMETRY, a="geometry")
     addmxPointNode(gemotryNode, 'mxPoint', a="sourcePoint",
                    x="0.0", y="11.0")
     ArrayNode = addArray(gemotryNode, TYPE_ARRAY, a="points")
     addmxPointNode(gemotryNode, 'mxPoint', a="targetPoint",
                    x="10.0", y="-4.0")
-    CCLink = addLink(root, TYPE_LINK, id="-63efee48:189fd5ed04e:-73d3",
-                     parent="128c18ea:1383ab8277e:-748d",
-                     source="-63efee48:189fd5ed04e:-73da",
-                     target="-63efee48:189fd5ed04e:-73d8",
+    CCLink = addLink(root, TYPE_LINK, id=link_id[2],
+                     parent=link_id[3],
```

Comment:
```suggestion
                     parent=block_id[0],
```
replace with block_id[0]